im.pidgin.Pidgin3.yml

```diff
@@ -0,0 +1,126 @@
+  - "--env=PIDGIN_PLUGIN_PATH=/app/plugins/pidgin-3"
+
+add-extensions:
+  im.pidgin.Purple3.Plugin:
+    directory: plugins
+    add-ld-path: lib
+    merge-dirs: purple-3
+    no-autodownload: true
+    autodelete: true
+    subdirectories: true
+  im.pidgin.Pidgin3.Plugin:
+    directory: plugins
+    add-ld-path: lib
+    merge-dirs: pidgin-3
+    no-autodownload: true
+    autodelete: true
+    subdirectories: true
+
+modules:
+  - name: libidn
+    buildsystem: autotools
+    cleanup:
+      - /lib/*.la
+    config-opts:
+      - "--disable-static"
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/libidn/libidn-1.42.tar.gz
+        sha256: d6c199dcd806e4fe279360cb4b08349a0d39560ed548ffd1ccadda8cdecb4723
+  - name: birb
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=false"
+      - "--wrap-mode=nofallback"
+    sources:
+      - type: archive
+        archive-type: tar-xz
+        url: https://sourceforge.net/projects/pidgin/files/birb/0.2.0/birb-0.2.0-dev.tar.xz/download
+        sha256: b4c24426a6c9aeb959a7a8d60d31def8e90cfb17680c90a0cc54d582789ecc38
+  - name: hasl
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=false"
+      - "--wrap-mode=nofallback"
+    sources:
+      - type: archive
+        archive-type: tar-xz
+        url: https://sourceforge.net/projects/pidgin/files/hasl/0.3.2/hasl-0.3.2.tar.xz/download
+        sha256: d67ba1ce29c6f1fcdc69dadae0bafe12cf60b9e8be80bb67b5f4f9e0db44c427
+  - name: ibis
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=false"
+      - "--wrap-mode=nofallback"
+    sources:
+      - type: archive
+        archive-type: tar-xz
+        url: https://sourceforge.net/projects/pidgin/files/ibis/0.10.1/ibis-0.10.1.tar.xz/download
+        sha256: 3c6a43a5cbbf80efa91e90e5b62e19e37258e788734feee76ab6fb6b7e13bb8c
+  - name: xeme
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=false"
+      - "--wrap-mode=nofallback"
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://keep.imfreedom.org/xeme/xeme/archive/5d0707ab10a2.tar.gz
+        sha256: eabb0f88fdc54775e9aab63d277d0d0cf54c456ee09580d45bd516d1117088da
+  - name: gplugin
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=false"
+      - "-Dlua=false"
+      - "-Dpython3=false"
+      - "--wrap-mode=nofallback"
+    sources:
+      - type: archive
+        archive-type: tar-xz
+        url: https://sourceforge.net/projects/pidgin/files/gplugin/0.44.2/gplugin-0.44.2.tar.xz/download
+        sha256: aea244e1add9628b50ec042c54cf93803f4577f8f142678f09b91fd4c0b20f72
+  - name: pidgin3
+    buildsystem: meson
+    config-opts:
+      - "-Ddoc=false"
+      - "--wrap-mode=nofallback"
+    sources:
+      - type: archive
+        archive-type: tar-gzip
+        url: https://keep.imfreedom.org/pidgin/pidgin/archive/7cd7744de3bc.tar.gz
+        sha256: 99089364aec61244f716c5fe2dc787dd724a07cb2021dd959901c7d62d0db55e
```

Comment:
how often is this going to be updated until it reaches a stable release? We generally don't allow nightly releases ie. that advertises itself has nightly or has a release cadence like that.

Reply:
We're not planning on doing nightlies at all, I just needed something to grab before the experimental 1 release (2.90.0) that we're releasing on December 31st.